class/defaults.yml

```
@@ -0,0 +1,20 @@
+parameters:
+  grafana_organizations_operator:
+    =_metadata: {}
+    namespace: vshn-appuio-grafana
+    grafana_secret: grafana-admin-password
+    grafana_username: ""
+    grafana_password: ""
+    grafana_url: http://vshn-appuio-grafana/
+    control_api_url: https://api.appuio.cloud/
+    control_api_token: ""
```

Best practice would be to default to secret references for secret values:

```suggestion
    grafana_password: ?{vaultkv:${cluster:tenant}/${cluster-name}/grafana-organizations-operator/grafana-password}
    grafana_url: http://vshn-appuio-grafana/
    control_api_url: https://api.appuio.cloud/
    control_api_token: ?{vaultkv:${cluster:tenant}/${cluster:name}/grafana-organizations-operator/control-api-token}
```

---

OK, I've done this for the control_api_token, but this doesn't work for the grafana_password. The latter doesn't need to be set at all, because we read the password from the secret "grafana-admin-password" (which was created by the Grafana Helm chart) by default and setting a grafana_password would override that and break the mechanism.